schema.sql

```diff
@@ -0,0 +1,10 @@
+/* Database schema to keep the structure of entire database. */
+
+CREATE TABLE animals(
+  id      		  	INT,
+  name    			  VARCHAR(250),
+  date_of_birth   DATE,
+  escape_attempts INT,
+  neutered     		BIT,
```

Comment:
- [ ] Good job so far 👍 Kindly make the `neutered` column `boolean` and not `bit` so as to follow the project requirements 😄